x-pack/legacy/plugins/maps/public/layers/styles/vector/style_util.js

```diff
@@ -34,22 +34,6 @@ export function isOnlySingleFeatureType(featureType, supportedFeatures, hasFeatu
   }, true);
 }
 
-export function scaleValue(value, range) {
```

Comment:
I still think we need to preserve this logic and snap NaN values to less then min so they have the same behavior and do not cause errors. Also, snapping values less then min and over max need to be preserved. When using aggregations to fetch min/max, there will be values greater than and less then min/max

Reply:
thanks, agreed.

After discussion, we're moving this clamping to the actual mb-expression. This way, we do not need to retain this separate `scaleValue` code-path. It has following template: https://github.com/elastic/kibana/pull/58528/files#diff-447d2991fd82818ee09399425def699eR67